files/image_config/logrotate/logrotateOverride.conf

```diff
@@ -1,2 +1,6 @@
 [Unit]
-Requires=logrotate-config.service
\ No newline at end of file
+Requires=logrotate-config.service
+
+[Service]
+ExecStart=
```

Comment:
@assrinivasan remove?

Reply:
No @prgeor. We need this to have logrotate write its status to /dev/shm/logrotate.

Reply:
The `ExecStart=` line is intentional. It's to tell systemd to clear out the previous value set for `ExecStart`. Without this line, systemd will append the new command, and the service will have two commands to run; in this case, logrotate would then run twice.